<launch>
    <arg name="launch_prefix" default="" />

    <param name="robot_name" value="phobos"/>
    <node pkg="gone" type="orchestrator_node.py" name="ground_orchestrator" output="screen">
        <param name="check_rate" value="0.1"/>
    </node>

    <node pkg="gone" type="monitor_node.py" name ="ground_monitor" output="screen"/>
    <node pkg="dtc_scripts" type="xbee_reciever.py" name="xbee_estop" output="screen"/>

</launch>
